### PR TITLE
ci: bot identity correction

### PR DESCRIPTION
**Pull request by lucasfcnunes**
## What type of PR is this? 

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] New Query Runner (Data Source) 
- [ ] New Alert Destination
- [ ] Other

## Description

Fix no identity on bot commits:

**now**

image

**expected**

image

PS:
- some autoformatting was done
- `${date}` -> `${TAG_NAME}`

## How is this tested?

- [ ] Unit tests (pytest, jest)
- [ ] E2E Tests (Cypress)
- [ ] Manually
- [x] N/A

## Related Tickets & Documents

- https://github.com/actions/checkout/discussions/479

### CODE REVIEW

File: .github/workflows/periodic-snapshot.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Periodic Snapshot
 # 10 minutes after midnight on the first of every month
 on:
   schedule:
-  - cron: "10 0 1 * *"
+    - cron: '10 0 1 * *'
```
Comment (lucydodo) [inline .github/workflows/periodic-snapshot.yml:6]: Just out of curiosity: Was this an accidental change by Linter? Or was it intentional?

File: .github/workflows/periodic-snapshot.yml [hunk 2]
```diff
@@ -16,13 +16,14 @@ jobs:
...
           ssh-key: ${{secrets.ACTION_PUSH_KEY}}
       - run: |
-          date="$(date +%y.%m).0-dev"
-          gawk -i inplace -F: -v q=\" -v tag=$date '/^  "version": / { print $1 FS, q tag q ","; next} { print }' package.json
-          gawk -i inplace -F= -v q=\" -v tag=$date '/^__version__ =/ { print $1 FS, q tag q; next} { print }' redash/__init__.py
-          gawk -i inplace -F= -v q=\" -v tag=$date '/^version =/ { print $1 FS, q tag q; next} { print }' pyproject.toml
-          git config user.name github-actions
-          git config user.email github-actions@github.com
+          git config user.name 'github-actions[bot]'
+          git config user.email 'github-actions[bot]@users.noreply.github.com'
```
Comment (lucydodo) [inline .github/workflows/periodic-snapshot.yml:20]: In my experience, `github-actions[bot]` using the following email address.
> 41898282+github-actions[bot]@users.noreply.github.com

For reference:
- https://github.com/sqlitebrowser/homebrew-sqlite3/pull/1#discussion_r1148304502
- https://github.com/orgs/community/discussions/26560

File: .github/workflows/periodic-snapshot.yml (hunk 1 above)
Comment (lucasfcnunes) [reply .github/workflows/periodic-snapshot.yml:6]: intentional:

> - some autoformatting was done

File: .github/workflows/periodic-snapshot.yml (hunk 2 above)
Comment (lucasfcnunes) [reply .github/workflows/periodic-snapshot.yml:20]: Seems that it works without the long numerical id.

> - https://github.com/actions/checkout/discussions/479

Which one it better?

File: .github/workflows/periodic-snapshot.yml (hunk 1 above)
Comment (lucydodo) [reply .github/workflows/periodic-snapshot.yml:6]: Oh, I missed that, thanks. :)

**Review by lucydodo — DISMISSED** [review]

File: .github/workflows/periodic-snapshot.yml (hunk 2 above)
Comment (lucasfcnunes) [reply .github/workflows/periodic-snapshot.yml]: made the change

**Review by justinclift — APPROVED** [review]
Cool, this looks like a good improvement. :smile:

**Comment by justinclift** [comment]
Thanks for this @lucasfcnunes, and thanks @lucydodo for reviewing it. :smile:

**Pull request merged**